internal/pkg/manifest/svc.go

```diff
@@ -513,19 +513,28 @@ func (cfg NetworkLoadBalancerListener) exposedPorts(exposedPorts []ExposedPort,
 	if cfg.TargetPort != nil {
 		targetPort = uint16(aws.IntValue(cfg.TargetPort))
 	}
+	targetProtocol := strings.ToLower(TCP)
+	if nlbProtocol != nil {
+		protocol := strings.ToLower(aws.StringValue(nlbProtocol))
+		// Expose TCP port for TLS listeners.
+		if protocol != strings.ToLower(TLS) {
+			targetProtocol = protocol
+		}
```

Comment:
Yay thanks for taking care of this case!!


Small nit - `strings.EqualFold` is case-insensitive, so we can use it for the comparison here
```suggestion
		// Expose TCP port for TLS listeners.
		if !strings.EqualFold(aws.StringValue(nlbProtocol), TLS)
			targetProtocol = protocol
		}
```

Reply:
TIL! Will update to use this